library/src/com/jeremyfeinstein/slidingmenu/lib/CustomViewAbove.java

```diff
@@ -4,13 +4,6 @@
 import android.graphics.Canvas;
 import android.graphics.Rect;
 import android.graphics.drawable.Drawable;
-import android.os.Build;
```

Comment:
I think it is not ideal to use an internal formatter on forked libraries. it will mean large changes in every affected class if we don't use a worldwide standard formatter.